### ISSUE TITLE
TracingRequestListener - nonexistent service

### ISSUE THREAD

**Issue by Glorian** [Baldinof/roadrunner-bundle#106]
Hi, encountered an issue with PR #67.

If the configuration parameter `tracing.enable = false`, then the container cannot be built because of this error:

```
 The service "Baldinof\RoadRunnerBundle\Integration\Sentry\SentryTracingRequestListenerDecorator" has a dependency on a non-existent service "Sentry\SentryBundle\EventListener\TracingRequestListener".
```

SentryExtension does not build TracingRequestListener service if this option is disabled.

**Comment by Baldinof** [comment]
Hello @Glorian!

Thanks for reporting, can you try #107 to validate it's working for you?

**Comment by Glorian** [comment]
Hi @Baldinof!
Yes, everything is working perfectly now! Thanks

**Comment by Baldinof** [comment]
Alright, it's released with `v2.3.2`.

**Comment by iborysenko** [comment]
Thanks @Baldinof for these fix
